controllers/utils/conditions.go

```diff
@@ -82,6 +82,20 @@ var ConditionReasons = struct {
 	InvalidTransition: "InvalidTransition",
 }
 
+// Condition messages
```

Comment:
Maybe note that this is just the set of common messages, not the complete set, as the message field is a freeform string that is also used for other status messages and relaying errors. ie. the message field can contain other values besides just this set

Reply:
Sure! I considered having these common messages in a conditionMessage struct but thought it might cause confusion as we have other custom messages. Updated the comment.